source/includes/fact-connect-intro.rst

```diff
@@ -0,0 +1,7 @@
+The :ref:`mongosync <c2c-mongosync>` utility creates a connection
+between MongoDB clusters. ``mongosync`` can make a connection between:
+
+- :ref:`self-managed clusters <server-replica-set-deploy-all>`,
+- :atlas:`MongoDB Atlas </>` hosted clusters, or 
+- a self-managed cluster and an Atlas hosted cluster.
```

Comment:
delete period at the end

Reply:
should we split these two up into separate bullets?

Reply:
The prior lines are

aaaaaa:
  xxx,
  yyy, or

The period is needed to close out the sentence.

Reply:
makes sense